drivers/src/measurement.rs

```diff
@@ -0,0 +1,29 @@
+/*++
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    measurement.rs
+
+Abstract:
+
+    Caliptra measurement related types.
+
+--*/
+use zerocopy::{AsBytes, FromBytes};
+
+pub const MEASUREMENT_MAX_COUNT: u32 = 8;
+
+/// Caliptra DPE Measurement    
+#[repr(C)]
+#[derive(AsBytes, FromBytes, Default, Debug)]
+pub struct StashMeasurement {
+    /// Checksum
+    pub checksum: u32,
+
+    /// Metadata
+    pub metadata: [u8; 4],
+
+    /// Measurement
+    pub measurement: [u32; 12],
+}
```

Comment:
This should rely on the structures in `mailbox_api.rs` to make sure we're using the same values: https://github.com/chipsalliance/caliptra-sw/blob/9cb9465c343da8c5c937d7269b32469644839735/common/src/mailbox_api.rs#L240

Reply:
Updated.